build-system/npm-publish/write-package-files.js

```diff
@@ -72,11 +72,13 @@ async function getHtmlTextContent(html) {
  * @return {Promise<?string>}
  */
 async function getFirstParagraphOrSentence(markdown, maxLengthChars) {
-  const token = marked.lexer(markdown).find(({type}) => type === 'paragraph');
+  const token = marked.Lexer.lex(markdown).find(
+    ({type}) => type === 'paragraph'
+  );
   if (!token) {
     return null;
   }
-  const html = marked.parser([token]);
+  const html = marked.Parser.parse([token]);
```

Comment:
No changes in functionality in this file. The newly installed type definitions for `marked` only know about this way of calling these functions, so we change it to prevent errors.